cohort-engine/src/test/java/com/ibm/cohort/engine/CqlTemporalTests.java

```diff
@@ -377,4 +378,25 @@ public Encounter getEncounter(int year, int month, int day) {
 		encounter.setPeriod(encounterPeriod);
 		return encounter;
 	}
+
+	private CqlEngineWrapper setupTestFor(Patient patient, FhirServerConfig fhirConfig, String... elm)
```

Comment:
I'd like to see you move the setupTestFor(Patient...) methods from CqlEngineWrapper into BaseFhirTest since you need them vs. copy/pasting.

Reply:
I was going back and forth on that one, since I feel like that setup might be different as we add different tests, but I can change.